subtype/src/test/java/com/fasterxml/jackson/module/subtype/WithoutJsonSubTypesTest.java

```diff
@@ -0,0 +1,121 @@
+package com.fasterxml.jackson.module.subtype;
+
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.auto.service.AutoService;
+import org.junit.Test;
+
+import java.util.Objects;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * test work without {@link JsonSubTypes}
```

Comment:
I am not sure I understand -- why would we test functioning without module? And isn't module actually register with the mapper. Or what is being tested here? Could probably use longer explanation of reasoning.

Reply:
Probably test against Jackson polymorphic type handling working as expected. If so, does not seem necessary

Reply:
I actually wanted to test whether `@JsonSubType` worked with `@JsonSubTypes`